temporalio/converter.py

```diff
@@ -209,10 +262,14 @@ def to_payloads(
             # We intentionally attempt these serially just in case a stateful
             # converter may rely on the previous values
             payload = None
-            for converter in self.converters.values():
-                payload = converter.to_payload(value)
-                if payload is not None:
-                    break
+            # RawValue should just pass through
+            if isinstance(value, temporalio.common.RawValue):
+                payload = value.payload
+            else:
+                for converter in self.converters.values():
+                    payload = converter.to_payload(value)
+                    if payload is not None:
+                        break
             if payload is None:
                 raise RuntimeError(
                     f"Value at index {index} of type {type(value)} has no known converter"
```

Comment:
This just code golf but FWIW this function has probably gained enough complexity now that it would be more readable as a generator:

```python
        return list(self._payloads(values))

    def _payloads(
        self, values Sequence[Any]
    ) -> List[temporalio.api.common.v1.Payload]:
        for index, value in enumerate(values):
            # We intentionally attempt these serially just in case a stateful
            # converter may rely on the previous values
            # RawValue should just pass through
            if isinstance(value, temporalio.common.RawValue):
                yield value.payload
                continue
            else:
                for converter in self.converters.values():
                    payload = converter.to_payload(value)
                    if payload is not None:
                        yield payload
                        continue
            raise RuntimeError(
                f"Value at index {index} of type {type(value)} has no known converter"
            )
```

Reply:
Even if the complexity justifies changing to a generator (unsure that it does tbh), this is an implementation of an abstract method we cannot change the signature of.

Reply:
Not sure this small function deserves to be delegated to another small function. I don't think it's that complicated. Also, your second continue needs to be a break.